Dockerfile.QA

```diff
@@ -333,7 +333,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 RUN rm -f /usr/bin/python && \
     ln -s /usr/bin/python3 /usr/bin/python
 
-RUN pip3 install --upgrade wheel setuptools && \
+RUN pip3 install --upgrade wheel setuptools packaging && \
```

Comment:
Shouldn't `packaging` be added to client-side `requirements_grpc.txt` instead? I believe users doing `pip install tritonclient[grpc]` outside of the SDK container will not have the `packaging` package and likely hit an import error.